docs/tutorials/02_create_your_first_app/05_storing-data.md

```diff
@@ -74,13 +74,14 @@ Here's the code with comments inline:
     };
 
 ```
+
 ### What this does
-- The `getVotes` method returns an `Array`. [`Array`](../../motoko/main/base/Array.md) is a simple container that can hold multiple values.
-- In this particular case, the array holds elements that is a tuple. The tuples in this case are of type `(Text, Nat)`
-- We use a class `Iter` that represents an iterator, a pointer-like data structure that allows the developer to move over another data structure and see values one by one in a sequential manner.
+- The `getVotes` method returns an `Array`. [`An array`](../../motoko/main/base/Array.md) is a simple container that can hold multiple values.
```

Comment:
[`An array`] looks odd to my eyes but not opinionated

Reply:
Good call! I removed the redundant 'an array' bit and combined the two sentences into one so its a little better on the eyes.